infrastructure/lambda-pdf-stitching.tf

```
@@ -20,6 +20,7 @@ module "pdf-stitching-lambda" {
   api_execution_arn       = null
   is_invoked_from_gateway = false
   lambda_environment_variables = {
+    APIM_API_URL                          = data.aws_ssm_parameter.apim_url.value
```

Since the ssm_parameter resource is named `apim_url`, I would suggest naming the environment variable likewise.
```suggestion
    APIM_URL                          = data.aws_ssm_parameter.apim_url.value
```

---

I would agree if it was a new var, however it is already named `APIM_API_URL` elsewhere